source/sass/image-picker.scss

```diff
@@ -22,25 +22,32 @@ ul.thumbnails.image_picker_selector
     margin:               0px;
   }
 
+  li.group
+  {
+    width: 100%
+  }
+
   li.group_title
   {
     float: none;
   }
 
   li
   {
-    margin:   0px $thumbnail_margin $thumbnail_margin 0px;
+    margin:   4px $thumbnail_margin $thumbnail_margin 4px;
     float:    left;
 
     .thumbnail
     {
       padding:  $thumbnail_padding;
       border:   $thumbnail_border;
+      border-radius: 4px;
       -webkit-user-select: none;
       -moz-user-select:    none;
       -ms-user-select:     none;
       img
       {
+        display:block;
```

Comment:
Why is this necessary?

Reply:
Not sure, probably isn't needed. @lowegreg is this needed for any reason?

Reply:
The img display is a little more important.  If it's not block the image doesn't align vertically in the middle of the thumbnail div.  Leaving a larger blue selected area on the bottom.  Only a few pixels, but noticeable...

Reply:
Could this be `inline-block` then? so it behaves like a normal image.

Reply:
You could use inline-block if you adjust the line-height of the ul.thumbnails.image_picker_selector to 0px.  The img has an implicit line-height assigned to it. Since line-height only applies to inline elements, setting the img to display: block removes the line-height value.

Reply:
Any decision on which way to go with this?